prow/scripts/cluster-integration/helpers/delete-dns-record.sh

```diff
@@ -24,8 +24,8 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
-attempts=3
-retryTimeInSec="5"
+attempts=10
+retryTimeInSec="10"
```

Comment:
Static wait time will not solve this. Please implement a random number generation that will get a number between 5 and 15 seconds

Reply:
Changed to randomizing the wait time from 5 to 14 secs.